src/Blazored.Modal/BlazoredModalInstance.razor

```diff
@@ -1,8 +1,10 @@
 ﻿@if (UseCustomLayout)
 {
-    <CascadingValue Value="this" IsFixed="true">
-        @Content
-    </CascadingValue>
+    <div>
+        <CascadingValue Value="this" IsFixed="true">
+            @Content
+        </CascadingValue>
+    </div>
```

Comment:
Why is the `div` tag needed?

Reply:
This is needed for the css to be applied to CustomLayouts.

Reply:
In what way? Just when it's removed everything looks to continue working as it did before and there don't seem to be any references to this element in the CSS, unless I've missed it?

Reply:
@chrissainty 
About the div, again...

This is **without** the div:
https://github.com/Blazored/Modal/assets/8792259/b1a629dd-fad1-4f51-9284-cc8765527ad1

This is **with** the div
https://github.com/Blazored/Modal/assets/8792259/4d12827a-e820-425f-bb6d-7191d8bcb945

Why does this happen? 
Because of the [Blazor CSS isolation](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/css-isolation?view=aspnetcore-8.0#child-component-support)
Without this change the custom layout that we have for the modals wont have the custom `BlazoredModalInstance.razor.css` styles applied...
I also had to add the following selectors on the `BlazoredModalInstance.razor.css`:
![image](https://github.com/Blazored/Modal/assets/8792259/4f0bf67b-02da-4201-b7c4-eab3858b2dee)